src/main/java/com/gluonhq/substrate/util/ios/InfoPlist.java

```diff
@@ -96,13 +96,24 @@ public InfoPlist(ProcessPaths paths, InternalProjectConfiguration projectConfigu
     }
 
     public Path processInfoPlist() throws IOException {
-        Path workDir = appPath;
-        String executableName = getExecutableName(projectConfiguration.getAppName(), sourceOS);
+        String appName = projectConfiguration.getAppName();
+        String executableName = getExecutableName(appName, sourceOS);
+        Path executable = appPath.resolve(executableName);
+        if (!Files.exists(executable)) {
+            String errorMessage = "The executable " + executable + " doesn't exist.";
+            if (!appName.equals(executableName) && Files.exists(appPath.resolve(appName))) {
+                errorMessage += "\nMake sure the CFBundleExecutable key in the Default-Info.plist file is set to: " + appName;
```

Comment:
where is that Default-Info.plist? In a src/gensrc/target dir? Make sure that if there are multiple ones (e.g. copied or generated) the user modifies the correct one.

Reply:
Fixed